packages/api/src/store/stream-table.ts

```diff
@@ -382,6 +382,7 @@ export default class StreamTable extends Table<DBStream> {
       transcodedSegmentsDuration: 0,
       sourceBytes: 0,
       transcodedBytes: 0,
+      lastTerminatedAt: null,
```

Comment:
@victorges the line below `...obj` will overwrite any of these default values if `obj` has the params set right?

Reply:
Exactly